.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -97,6 +97,15 @@ jobs:
           #  encryption: 'ipsec'
           #  endpoint-routes: 'true'
 
+          - config: '6.1'
+            # renovate: datasource=docker depName=quay.io/lvh-images/kind
+            kernel: '6.1-20231026.065108'
+            kube-proxy: 'iptables'
+            kpr: 'disabled'
+            tunnel: 'vxlan'
+            encryption: 'ipsec'
+            endpoint-routes: 'false'
+
```

Comment:
@nbusseneau Could you append the following to the commit which added this config?
```
# https://github.com/cilium/cilium/issues/29805#issuecomment-1855600171
ipv6: 'false'
```